Makefile

```diff
@@ -141,6 +141,10 @@ k8s: validate ## Build default K8s version of EKS Optimized AL2 AMI
 1.28: ## Build EKS Optimized AL2 AMI - K8s 1.28
 	$(MAKE) k8s $(shell hack/latest-binaries.sh 1.28)
 
+.PHONY: 1.29
+1.29: ## Build EKS Optimized AL2 AMI - K8s 1.29
+	$(MAKE) k8s $(shell hack/latest-binaries.sh 1.29)
+
 .PHONY: lint-docs
```

Comment:
We're not adding stub targets for every version going forward, you can do:
```
make k8s=1.29
```

Reply:
@cartermckinnon  Understood. Do you want me to remove others as well?

Reply:
We don't want to break anyone by removing them, they'll just go away as the k8s versions fall out of support 👍

Reply:
Alright, in my most recent commit I removed _only_ the target added by me, i.e. 1.29.